tests/data/placebo/test_ecs_service_update/ecs.DescribeServices_2.json

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
afaict this file is extra and can be removed, since this service isn't in the output of list services or the one thats being modified.

Reply:
When I remove the file tests/data/placebo/test_ecs_service_update/ecs.DescribeServices_2.json the test seems to fail with:

FAIL: test_ecs_service_update (tests.test_ecs.TestEcsService)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/WNZ812/repo/cloud-custodian-egunnar-github/tests/test_ecs.py", line 141, in test_ecs_service_update
    'awsvpcConfiguration']['assignPublicIp'], 'DISABLED')
AssertionError: 'ENABLED' != 'DISABLED'
- ENABLED
+ DISABLED

Reply:
Although I had difficulty removing this file I was able to significantly reduce it's size. I applied your earlier advice about removing the contents of the events section to it.

Reply:
thanks for trying, and also for for reducing the size, i'd like to take a look as well as it makes me wonder whats causing the additional api call, it could be something outside of this pr.

Reply:
it removed cleanly for me, removing the second describe, also means renaming the third. pushed as an update to the branch.